src/components/explorer/IndividualTracks.js

```diff
@@ -165,7 +165,12 @@ const IndividualTracks = ({ individual }) => {
         referenceGenomes.items.forEach((g) => {
             availableGenomes[g.id] = { id: g.id, fastaURL: g.fasta, indexURL: g.fai };
         });
-        (igvGenomes.data ?? []).forEach((g) => availableGenomes[g.id] = g);
+        (igvGenomes.data ?? []).forEach((g) => {
+            availableGenomes[g.id] = g;
+            // Manual aliasing for well-known human genome aliases:
+            if (g.id === "hg19") availableGenomes["GRCh37"] = {...g, id: "GRCh37"};
+            else if (g.id === "hg38") availableGenomes["GRCh38"] = {...g, id: "GRCh38"};
```

Comment:
Would it be worth it to have the reference service store these aliases instead? As we support non-human genomes the list of aliases could grow. 
Alternatively, we could have some sort of genomes_aliases.json config file instead of hard coding here.

Reply:
the ref service can store aliases; this is because for now we want to take advantage of igv.js's gene tracks, but they only store GRCh37/38 as hg19/38. In the future, this can be removed since we will ingest GFF3 files into the reference service. I'll add a comment explaining this

Reply:
Can report that for mice in particular using "mm10" as an assembly gets me the "RefSeq Genes" track from igv, but using "GRCm38" doesn't, even though they're the same assembly.

Reply:
ah i will add that then and see if igv.js has a built-in list of aliases